packages/hdwallet-metamask-multichain/src/shapeshift-multichain.ts

```diff
@@ -94,7 +96,7 @@ export class MetaMaskShapeShiftMultiChainHDWalletInfo implements core.HDWalletIn
       case "atom":
         return core.cosmosDescribePath(msg.path);
 
-      case "ethereum":
```

Comment:
really feels like we should do this to all of the coins in here to make sure they match the `slip44Table` (not that we really use this function anywhere, but for posterity)

Reply:
All in for cleanup! e0e8f4e587c7b8143c9603066ad92ab08fe08762